src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -832,4 +838,166 @@ public void cannotCreateARequestWithUnknownPickupLocationServicePoint()
       hasMessage("Pickup service point does not exist"),
       hasUUIDParameter("pickupServicePointId", pickupServicePointId))));
   }
+
+  @Test
+  public void CanCreatePagedRequestWhenItemStatusIsAvailable()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException{
+
+    //Set up the item's initial status to be AVAILABLE
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final String itemInitialStatus = smallAngryPlanet.getResponse().getJson().getJsonObject("status").getString("name");
+    assertThat(itemInitialStatus, CoreMatchers.is(ItemStatus.AVAILABLE.getValue()));
+
+    //Attempt to create a page request on it.  Final expected status is PAGED
+    final IndividualResource servicePoint = servicePointsFixture.cd1();
+    final IndividualResource pagedRequest = requestsClient.create(new RequestBuilder()
+      .page()
+      .forItem(smallAngryPlanet)
+      .withPickupServicePointId(servicePoint.getId())
+      .by(usersFixture.james()));
+
+    String finalStatus = pagedRequest.getResponse().getJson().getJsonObject("item").getString("status");
+    assertThat(pagedRequest.getResponse(), hasStatus(HTTP_CREATED));
+    assertThat(finalStatus, CoreMatchers.is(ItemStatus.PAGED.getValue()));
```

Comment:
I find it improves clarity of asserts to statically import is matcher.
```suggestion
    assertThat(finalStatus, is(ItemStatus.PAGED.getValue()));
```